### ISSUE TITLE
OLA and Swiggy

### ISSUE THREAD

**Issue by naveenpf** [piebro/OpenStreetMap-Statistics#113]
please add 
https://wiki.openstreetmap.org/wiki/Ola
https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities/Swiggy 
https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities/Stackbox

**Comment by piebro** [comment]
I just added Ola and Stackbox was already present. While adding Swiggy I made an interesting observation.
After running my code it seemed there were no edits from Swiggy users in the last month or earlier. I checked the history of some users of the Swiggy Corporation to check if my code is faulty. Some Swiggy users made some recent edits but also some more than three years ago. I was looking at my code and why those edits weren't counted and I saw that for example the changeset [79149054](https://www.openstreetmap.org/changeset/79149054) is done by the user swiggy002 but the changeset log lists another user:

```
c79149054 k7 s2020-01-03T09:50:27Z e2020-01-03T09:50:29Z d0 i10604190 uManitha%20%kumari%20%k x72.8216596 y18.9753421 X72.8245812 Y18.978379 Tcomment=Added%20%missing%20%%20%u-turn,created_by=iD%20%2.16.0,host=https://maps-columbus.production.singapore.swiggy.cloud/,imagery_used=Bing%20%aerial%20%imagery,locale=en-US
```

My most plausible explanation is that the user changed its name to swiggy002 and there the changeset is visible in the user history but it's different in the changeset log. The recent edits of Swiggy users should be in the statistics next month.
